templates/Backend.h

```diff
@@ -142,7 +142,7 @@ class Entity {
 		 * @tparam T Type of component to ask for.
 		 * @return Pointer to component of type T or nullptr.
 		 */
-		template<typename T> T* Get() {
+		template<typename T> const T* Get() const {
 			int index = detail::ComponentPriority<T>::value;
 			int offset = componentOffsets[index];
 			if (offset) {
```

Comment:
Change the cast from `T*` to `const T*` here? Otherwise looks good.

Reply:
Ah yes, overlooked that one. Thanks :-)